### PR TITLE
VMcollector: update documentation

### PR DESCRIPTION
**Pull request by Halamix2**
**Description**

Changes proposed in this pull request:

- Change regexp from include to exclude
- Tool now cleans up much more VMs than just the `kyma-integration` ones

**Related issue(s)**

#4718

### CODE REVIEW

**Comment by Halamix2** [comment]
/hold
Until #4718 is merged

File: development/tools/cmd/vmscollector/README.md
Comment (klaudiagrz) [inline development/tools/cmd/vmscollector/README.md:7]:
```suggestion
Prow jobs create a VM instance to install and test Kyma.
```

File: development/tools/cmd/vmscollector/README.md
Comment (klaudiagrz) [inline development/tools/cmd/vmscollector/README.md:15]:
```suggestion
- The value of the `job-name` label the instance is annotated with is not caught by the exclude labels regexp
```

File: development/tools/cmd/vmscollector/README.md
Comment (klaudiagrz) [inline development/tools/cmd/vmscollector/README.md:14]:
```suggestion
- The instance name is not caught by the exclude names regexp
```

File: development/tools/cmd/vmscollector/README.md
Comment (klaudiagrz) [inline development/tools/cmd/vmscollector/README.md:16]:
```suggestion
- The instance `creationTimestamp` value that is used to find instance existing at least for a preconfigured number of hours.
```

File: development/tools/cmd/vmscollector/README.md
Comment (klaudiagrz) [inline development/tools/cmd/vmscollector/README.md:14]:
```suggestion
- The instance name is not caught by the exclude names regexp.
```

File: development/tools/cmd/vmscollector/README.md
Comment (klaudiagrz) [inline development/tools/cmd/vmscollector/README.md:15]:
```suggestion
- The value of the `job-name` label the instance is annotated with is not caught by the exclude labels regexp.
```

File: development/tools/cmd/vmscollector/README.md
Comment (klaudiagrz) [inline development/tools/cmd/vmscollector/README.md:14]:
```suggestion
- The instance name is not caught by the exclude names regex.
```

Comment (klaudiagrz) [reply development/tools/cmd/vmscollector/README.md:14]: According to Google, it's `regex`, not `regexp` 🤔 
Could you verify and change it in the rest of the document if needed?

Comment (Halamix2) [reply development/tools/cmd/vmscollector/README.md]: Done

**Review by klaudiagrz — APPROVED** [review]

**Pull request merged**
